pkgs/applications/misc/keystore-explorer/default.nix

```
@@ -0,0 +1,28 @@
+{ fetchzip, stdenv, jre }:
+
+stdenv.mkDerivation rec {
+  version = "5.4.4";
+  name = "keystore-explorer-${version}";
```

`name` is deferred automatically by mkDerivation:

```suggestion
  pname = "keystore-explorer";
```